### PR TITLE
Brings authentication up-to-date

### PR DESCRIPTION
**Pull request by mfaux**
* Brings authentication up-to-date with the next version of Ace Platform.
* Synchronizes the `.env` files to make the authentication process identical for both samples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,58 @@
 # Configuration API Samples for Ace Platform
 
-This repository contains sample applications that show how to use Ace Platform's Configuration API.
```
Comment (mfaux) [inline README.md:3]: Changed contents to be 80 chars.

File: basics/README.md [hunk 2]
```diff
@@ -45,16 +37,20 @@ The project has the following structure:
     ├── api                 # helpers for calling the HTTP API
     ├── components          # general React components
     └── pages
-        ├── configurator    # example configurator using the /configure endpoint
+        ├── configurator    # example using the /configure endpoint
+        ├── pricing         # example using the /price endpoint
```
Comment (mfaux) [inline basics/README.md:41]: Added `pricing`

File: basics/docs/CONFIGURATOR.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ We start with a quick review of the `/configure` endpoint. The endpoint is state
 
 The `/configure` endpoint is designed for building interactive configurators. This means that:
 
-- The endpoint responds with a structure of `sections`, `variables` and `values`. This structure depends on the product we're configuring and a client application can use this structure to organize the the variables/values for the user.
```
Comment (mfaux) [inline basics/docs/CONFIGURATOR.md:28]: Fixes 'the the'

File: basics/docs/CONFIGURATOR.md [hunk 4]
```diff
@@ -696,7 +696,7 @@ function ConflictDialog({ conflict, onAccept, onReject }) {
 }
 ```
 
-The `<ConflictDialog>` expects a `conflict` object that has a `currentAssignment` and a list of `removedAssingments` and two functions or handling acceptance and rejection.
+The `<ConflictDialog>` expects a `conflict` object that has a `currentAssignment` and a list of `removedAssignments` and two functions or handling acceptance and rejection.
```
Comment (mfaux) [inline basics/docs/CONFIGURATOR.md:699]: Typo

File: basics/src/pages/configurator/components/Section.js [hunk 5]
```diff
@@ -41,7 +41,7 @@ export function Section({
           removedAssignments={removedAssignments}
         />
       ))}
-      {(section.sections || []).map(subSection => (
+      {(section.sections || []).map((subSection) => (
```
Comment (mfaux) [inline basics/src/pages/configurator/components/Section.js:44]: Hmm. Parens added on save.

File: basics/src/pages/configurator/components/Section.js [hunk 6]
```diff
@@ -26,13 +26,13 @@ export function Section({
...
     <section>
-      {/* The name of the first level of sections are already dislayed in the tabs */}
+      {/* The name of the first level of sections are already displayed in the tabs */}
       {level > 0 && (
         <header className={`section-header section-header-level${level}`}>
           {section.name}
         </header>
       )}
-      {visibleVariables.map(variable => (
+      {visibleVariables.map((variable) => (
```
Comment (mfaux) [inline basics/src/pages/configurator/components/Section.js:35]: Hmm. Parens added on save.

**Comment by mfaux** [comment]
fyi @kasperp , I did not run the code after my changes.

**Review by kasperp — APPROVED** [review]
Looks great, thanks

**Pull request merged**